terraform/aws/modules/eks/vpc.tf

```diff
@@ -0,0 +1,10 @@
+module "vpc" {
```

Comment:
while this works, i think it is better to keep the VPC module call in the example itself.

That way anyone using this can setup VPC themselves, and just pass the needed module configs to the EKS module. Else they would need to create a new VPC everytime they need to use this module.